### DISCUSSION TITLE
How to setting the repositories mirroring?

### DISCUSSION THREAD

**Discussion by YuryMikhailuts** [community/community#28287] [Apps, API and Webhooks]
Good afternoon, comrades. I have an open project repository on another git platform. For some reason, I want my repository to be mirrored on github as well. Can I set this up automatically?

**Comment by YuryMikhailuts** [comment]
I am grateful in advance for any answers.

**Comment by jfullstackdev** [comment]
maybe this one will help 

https://stackoverflow.com/questions/30268549/mirroring-from-gitlab-to-github
